lib/exports/colors-new/color-sets.less

```diff
@@ -0,0 +1,419 @@
+// White
```

Comment:
@dancormier Getting many style linting problems here about not known CSS properties in the mixins. My understanding is that we are using mixins in here to remedy the lack of better data structures in Less. Since this is intentional we can add this comment at the top of this file:
```
/* stylelint-disable property-no-unknown */
```